.github/workflows/nightly_build.yml

```diff
@@ -54,6 +54,30 @@ jobs:
         export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
         python3 kratos/python_scripts/run_tests.py -l nightly -c python3
 
+    - name: Running MPI Python MultilevelMonteCarloApplication (2 Cores)
```

Comment:
please don't modify the nightly script yet, I want to do it altogether

in any case the same tests are run in the CI already

Reply:
ah ok you only added it here, then it is ok